hack/Dockerfile

```diff
@@ -12,24 +12,55 @@ ENV PYTHONUNBUFFERED 1
 RUN set -ex \
 	&& apt-get update \
 	&& apt-get install -y --no-install-recommends \
+		atool \
 		apt-transport-https \
 		build-essential \
+		coreutils \
 		curl \
+		gcc \
 		gettext \
+		ghostscript \
 		git \
+		g++ \
 		gpg-agent \
+		hashdeep \
+		imagemagick \
+		inkscape \
 		libbz2-dev \
 		libffi-dev \
+		libldap2-dev \
 		liblzma-dev \
+		libimage-exiftool-perl \
+		libmysqlclient-dev \
 		libncursesw5-dev \
 		libreadline-dev \
+		libsasl2-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \
 		libxmlsec1-dev \
+		libxslt1-dev \
 		locales \
+		logapp \
+		md5deep \
+		mediaconch \
+		mediainfo \
+		nfs-common \
+		nginx \
+		openjdk-8-jre-headless \
+		p7zip-full \
+		pbzip2 \
+		postfix \
```

Comment:
Why are `ngnix` and `postfix` included? They were filtered out when `osdeps.py` was used:

```
/src/hack/osdeps.py Ubuntu-22 1 | grep -v -E "nginx|postfix|python3.9-dev"
```

Reply:
oh ya!